pombola/south_africa/templatetags/za_people_display.py

```diff
@@ -0,0 +1,29 @@
+from django import template
+
+register = template.Library()
+
+NO_PLACE_ORGS = ('parliament', 'national-assembly', )
+MEMBER_ORGS = ('parliament', 'national-assembly', )
+
+
+@register.assignment_tag()
+def should_display_place(organisation):
+    should_display = True
```

Comment:
I think it'd be easier to spot potential problems in this implementation if it were just the one-liner `return organisation.slug not in NO_PLACE_ORGS`